Fix project inline and styling

dalderliesten opened the pull request:
-Removed all inline comments that still existed.
-Made a general documentation and javadoc style across the project (spacing, style, etc.).
-Enhanced a lot of Javadoc.

Commit:
Remove Constants Inline

Removed the inline comments in the constants class.

<organization

Commit:
Fix Constants Javadoc

Fixed the exiting javadoc that was incorrectly written/implemented in
the constants class.  Added some javadoc for required classes.

<organization

Commit:
Fix Logger Inline

Removed a TODO inline comment from the logger class.

<organization, bug

Commit:
Fix StepTimer Inline

Fixed some empty lines within the step timer class.

<organization

Commit:
Fix Game Inline

Fixed some inline comments and spacing in the game class.

<organization

Commit:
Fix Gamefactory Space

Fixed some spacing in the game factory class.

<organization

Commit:
Fix mulitplayergame space

Fixed a spacing issue at the imports of the multiplayer game class.

<organization

Commit:
Fix singleplayergame space

Fixed some spacing issues in the singleplayergame class related to the
imports.

<organization

Commit:
Fix singleplayergamefactory Space

Fixed a random inline space in the single player game factory class.

<organization

Commit:
Fix keybinding spacing

Fixed some inline spaces in the keybinding and keybindingcontainer
classes.

<organization

Commit:
Fix level barebones inline

Removed numerous random spaces and inline comments from the level
package barebones classes.  Also added the required checkstyle
exceptions.

<organization

Commit:
Fix level package remain

Fixed the inline comments and spacing issues/javadoc issues in the rest
of the level package.

<organization, bug

Commit:
Fix powerup inline

Fixed inline comments and spacing issues in the powerup class.

<organization

Commit:
Fix spritestore inline

Fixed the sprite store inline spacing and comments.

<organization

dalderliesten committed:
Fix gameDisplay inline

Fixed some spacing issues and inline comments that existed in the gameDisplay class.

<organization

dalderliesten committed:
Fix keylistener spacing

Fixed some spacing issues in the keylistener class.

<organization

Commit:
Merge branch 'release' into remove-inline

Conflicts:
	src/main/java/nl/tudelft/scrumbledore/keybinding/Keybinding.java
	src/main/java/nl/tudelft/scrumbledore/level/Level.java
	src/main/java/nl/tudelft/scrumbledore/level/element/BasicDynamicElement.java
	src/main/java/nl/tudelft/scrumbledore/level/element/BasicStaticElement.java
	src/main/java/nl/tudelft/scrumbledore/level/element/Fruit.java
	src/main/java/nl/tudelft/scrumbledore/level/element/LevelElement.java
	src/main/java/nl/tudelft/scrumbledore/level/element/NPC.java
	src/main/java/nl/tudelft/scrumbledore/level/element/Platform.java
	src/main/java/nl/tudelft/scrumbledore/level/modifier/CollisionsLevelModifier.java
	src/main/java/nl/tudelft/scrumbledore/level/modifier/PlayerActionsLevelModifier.java
	src/main/java/nl/tudelft/scrumbledore/level/powerup/ChiliChickenPickUp.java
	src/main/java/nl/tudelft/scrumbledore/level/powerup/TurtleTacoPickUp.java
	src/main/java/nl/tudelft/scrumbledore/powerup/BlueberryBubble.java
	src/main/java/nl/tudelft/scrumbledore/powerup/BlueberryBubblePickUp.java
	src/main/java/nl/tudelft/scrumbledore/powerup/PyroPepper.java
	src/main/java/nl/tudelft/scrumbledore/powerup/PyroPepperPickUp.java

Commit:
Fix level.modifier inline

Fixed the new issues that arose after the merge in the level.modifier
package.

<organization

Commit:
Fix level.element inline

Fixed numerous level.element inline spacing and comment issues.

<organization

Commit:
Fix remainder inline

Fixed the remaining inline comment and spacing issues in the keybinding,
level, and game packages.

<organization

NielsWarnars commented:
Please merge the current release branch into this branch as merge conflicts seem to occur. 
<review

dalderliesten commented:
My earlier merge must have broken it.  One moment.

<review, bug

Commit:
Merge branch 'release' into remove-inline

Conflicts:
	src/main/java/nl/tudelft/scrumbledore/level/modifier/CollisionsLevelModifier.java

Commit:
Fix Merge Issue

Fixed an issue from the merge conflict that caused the work to be
reverted in keybindingsettings.

<organization, bug

dalderliesten commented:
It should now be fixed, Niels.

<review
